test/functional/apps/discover/_discover.js

```diff
@@ -19,20 +18,20 @@ bdd.describe('discover app', function describeIndexTests() {
     // delete .kibana index and update configDoc
     return esClient.deleteAndUpdateConfigDoc({'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*'})
     .then(function loadkibanaIndexPattern() {
-      common.debug('load kibana index with default index pattern');
+      PageObjects.common.debug('load kibana index with default index pattern');
```

Comment:
I feel like it's OK to have a generic page object that surfaces functionality useful across test suites. So in that sense, I think it's OK to have this Common Page Object. But I do think we should extract any methods that don't touch `remote` or otherwise interact with the browser, similarly to how I extracted Try and Log.